help/gmcommands.txt

```diff
@@ -0,0 +1,123 @@
+ <- @@index|<<Back to Index>>@@
```

Comment:
Also, instead of filling the bulk of these out by hand, generate them from tmwa itself, it already has a --write-atcommand-file setting.